### DISCUSSION TITLE
why my file don't use my domain name to share??

### DISCUSSION THREAD

**Discussion by fusengine** [minio/minio#13338] [Q&A] [answered]
And I put my certificat buy in to dir my don't work to login and  show error **X509**.

if I don't use **MINIO_SERVER_URL**  this link put ip docker. Why if I want share my bucket is don't use my domain?

``
'http://test.mimi.test:<yourport>/foo.jpg'
``

but why your documentation is very difficult to use.

```docker-compte
version: '3.9'
services:
  minio:
    restart: always
    image: minio/minio:latest
    environment:
      - MINIO_ROOT_USER=roo
      - MINIO_ROOT_PASSWORD=root*
      - MINIO_BROWSER_REDIRECT_URL=https://s3.domain.com
      - MINIO_REGION_NAME=eu-central-1
      - MINIO_DOMAIN= https://s3.domain.com
      - MINIO_SERVER_URL= https://s3.domain.com
    command: minio server /data --console-address ":9001" --certs-dir "/certs_minio"
    ports:
      - 9000:9000
      - 9001:9001
    volumes:
      - minio_data:/data
      - minio_certs:/certs_minio

volumes:
  minio_data:
  minio_certs:
```

do you have solution? thanks

**Comment by harshavardhana** [comment, marked as answer]
> if I don't use **MINIO_SERVER_URL** this link put ip docker. Why if I want share my bucket is don't use my domain?

If you want a custom domain for connectivity you need to specify MINIO_SERVER_URL - we cannot just assume that is the domain you want to do. 

https://github.com/minio/minio#things-to-consider talks about 
https://docs.min.io/minio/baremetal/reference/minio-server/minio-server.html#envvar.MINIO_SERVER_URL

**Reply by fusengine** [reply]
don't work I try this. the hold mini is very simple but new mini wow, a documentation don't good, exemple don't good, why mini complicated deployment is for buy a licence?

**Reply by fusengine** [reply]
why closed is don't resolve
